client/extensions/woocommerce/app/order/order-customer/create.js

```diff
@@ -173,10 +199,15 @@ export default connect(
 		const siteId = getSelectedSiteId( state );
 		const order = getOrderWithEdits( state );
 
+		const loadedLocations = areLocationsLoaded( state, siteId );
+		const countries = getAllCountries( state, siteId );
+
 		return {
+			countries,
```

Comment:
These two new props could also be added to `propTypes` for this component.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/22838/commits/705b284462b5390c6fa399f6c7eaef08f0cc96a9